e2e/deployers/appset.go

```diff
@@ -65,7 +66,11 @@ func (a ApplicationSet) Undeploy(ctx types.Context) error {
 
 	clusterName, err := util.GetCurrentCluster(util.Ctx.Hub, managementNamespace, name)
 	if err != nil {
-		return err
+		if !errors.IsNotFound(err) {
+			return err
+		}
```

Comment:
I think this call was removed in main, so this change is not needed.

But we may need the same change when calling GetCurrentCluster before DeleteApplicationSet(), since the application may already be deleted.

Lets start by rebasing on main.

Reply:
rebased, made changes and tested. Updated in description.